rockspec/lua-resty-hmac-ffi-1.0-0.rockspec

```diff
@@ -0,0 +1,24 @@
+package = 'lua-resty-hmac-ffi'
+version = '1.0-0'
+
+source = {
+  url = 'git://github.com/jamesmarlowe/lua-resty-hmac',
+  tag = 'v1.0'
+}
+
+description = {
+  summary = 'Lua driver for making and receiving hmac signed requests',
+  homepage = 'https://github.com/jamesmarlowe/lua-resty-hmac',
+  license = 'BSD-2-Clause License'
+}
+
+dependencies = {
+  'lua == 5.1'
+}
```

Comment:
I think we'll also need `https://github.com/openresty/lua-resty-string` somehow included -- not sure if that is published on luarocks.

Reply:
the `lua-resty-string` has been shipped with `openresty` by default.